### PR TITLE
Data Prepper 2.0.0 Release notes 

### LINKED ISSUE #1844: Create Data Prepper 2.0.0 Release Notes
Create Data Prepper 2.0.0 Release Notes

All changes should be available at:

https://github.com/opensearch-project/data-prepper/milestone/3?closed=1

### PR DESCRIPTION
**Pull request by asifsmohammed**
### Issues Resolved
resolves #1844 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed with a real name per the DCO

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+---
+
+### Features
+* Support conditional routing in sinks ([#1337](https://github.com/opensearch-project/data-prepper/issues/1337))
+* Support Core Peer Forwarder which replaces the peer forwarder processor plugin ([#700](https://github.com/opensearch-project/data-prepper/issues/700))
+* Support parsing CSV and TSV values in event fields using CSV processor ([#1081](https://github.com/opensearch-project/data-prepper/issues/1081))
+* Support processing JSON values in event fields using JSON processor ([#831](https://github.com/opensearch-project/data-prepper/issues/831))
+* Support multi node aggregation using Core Peer Forwarder ([#978](https://github.com/opensearch-project/data-prepper/issues/978))
+* Support peer forwarding in service_map_stateful processor using Core Peer Forwarder ([#1765](https://github.com/opensearch-project/data-prepper/issues/1765))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:11]: Maybe reword these to use "Use core peer forwarding in ... processor"

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Support peer forwarding in otel_trace_raw processor using Core Peer Forwarder ([#1766](https://github.com/opensearch-project/data-prepper/issues/1766))
+
+
+### Enhancements
+* Support concurrency in Data Prepper expression ([#1189](https://github.com/opensearch-project/data-prepper/issues/1189))
+* Support bulk create option in OpnSearch sink ([#1561](https://github.com/opensearch-project/data-prepper/pull/1561))
+* Added new metric to track utilization rate of buffer ([#1817](https://github.com/opensearch-project/data-prepper/issues/1817))
+* Support publishing metrics for Data Prepper core ([#1789](https://github.com/opensearch-project/data-prepper/issues/1789))
+* Support configurable timeouts for processors and sinks to flush the ata downstream before shutdown ([#1742](https://github.com/opensearch-project/data-prepper/issues/1742))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:20]: > the ata downstream

Do you mean "the data downstream" here?

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* Support for consistent decoding of s3 events ([#1687](https://github.com/opensearch-project/data-prepper/issues/1687))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:21]: Maybe rename this to:

> Moved the fields from S3 JSON objects up to the root level of the object to promote consistency with other S3 codecs.

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* Enabled HTTP health check for OTel trace source and OTel metrics source ([#1546](https://github.com/opensearch-project/data-prepper/issues/1546))
+* Made Java 11 to be the baseline version for Data Prepper ([#1422](https://github.com/opensearch-project/data-prepper/issues/1422))
+* Updated HTTP source request timeout to use configured timeout ([#975](https://github.com/opensearch-project/data-prepper/issues/975))
+* Distribute Data Prepper Docker image with JDK 17 ([#694](https://github.com/opensearch-project/data-prepper/issues/694))
+* Support ACM and S3 for TLS/SSL in HTTP source ([#365](https://github.com/opensearch-project/data-prepper/issues/365))
+* Updated HTTP methods supported on core API endpoints ([#313](https://github.com/opensearch-project/data-prepper/issues/313))
+
+### Bug Fixes
+* Fix a bug where file sink fails to write to output file with multiple pipeline threads ([#1843](https://github.com/opensearch-project/data-prepper/issues/1843))
+* Fixed health check bug when Auth is enabled for HTTP and OTel trace source ([#1600](https://github.com/opensearch-project/data-prepper/issues/1600))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:32]: I don't think this is was a bug. I'd consider this an enhancement to allow for an unauthenticated health-check endpoint.

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+### Maintenance
+* Removed support for OTLP protocol as internal data transfer in trace pipeline ([#1272](https://github.com/opensearch-project/data-prepper/issues/1272))
+* Update Jackson to 2.13.4 ([#1871](https://github.com/opensearch-project/data-prepper/pull/1871))
+* Updated Armeria to 1.19.0 ([#1806](https://github.com/opensearch-project/data-prepper/issues/1806))
+* Removed Peer Forwarder processor plugin ([#1874](https://github.com/opensearch-project/data-prepper/pull/1874))
+* Removed deprecated type property in DataPrepperPlugin annotation ([#1657](https://github.com/opensearch-project/data-prepper/issues/1657))
+* Removed deprecated trace_analytics_raw and trace_analytics_service_map values, index_type replaces them ([#1648](https://github.com/opensearch-project/data-prepper/issues/1648))
+* Removed deprecated PluginFactory ([#1584](https://github.com/opensearch-project/data-prepper/issues/1584))
+* Removed AWS SDK v1 from S3 source ([#1562](https://github.com/opensearch-project/data-prepper/issues/1562))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:49]: Perhaps:

> Removed AWS SDK v1 entirely from Data Prepper.

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+* Updated Antlr to 4.10.1 ([#1513](https://github.com/opensearch-project/data-prepper/issues/1513))
+* Updated all existing prepper plugins to use only processor. ([#647](https://github.com/opensearch-project/data-prepper/issues/647))
+* Removed deprecated prepper plugin type from pipeline configuration. ([#619](https://github.com/opensearch-project/data-prepper/issues/619))
+* Updated package naming from `com.amazon` to `org.opensearch` ([#344](https://github.com/opensearch-project/data-prepper/issues/344))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:53]: I'd suggest inverting the from and to here:

> to `org.opensearch` from `com.amazon`.

File: release/release-notes/data-prepper.release-notes-2.0.0.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+## 2022-10-23 Version 2.0.0
+
+---
+
+### Features
+* Support conditional routing in sinks ([#1337](https://github.com/opensearch-project/data-prepper/issues/1337))
+* Support Core Peer Forwarder which replaces the peer forwarder processor plugin ([#700](https://github.com/opensearch-project/data-prepper/issues/700))
+* Support parsing CSV and TSV values in event fields using CSV processor ([#1081](https://github.com/opensearch-project/data-prepper/issues/1081))
```
Comment (dlvenable) [inline release/release-notes/data-prepper.release-notes-2.0.0.md:8]: We also support directly decoding an S3 object as CSV which is a nice new feature. We should include that.

**Review by dlvenable — CHANGES_REQUESTED** [review]

File: release/release-notes/data-prepper.release-notes-2.0.0.md (hunk 2 above)
Comment (asifsmohammed) [reply release/release-notes/data-prepper.release-notes-2.0.0.md:20]: yes

**Review by dlvenable — APPROVED** [review]

**Review by oeyh — APPROVED** [review]
